src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -214,6 +214,11 @@ protected function describeContainerDefinition(Definition $definition, array $op
             }
         }
 
+        $calls = $definition->getMethodCalls();
+        foreach ($calls as $callData) {
+            $output .= "\n".'- Call: `'.$callData[0].'`';
```

Comment:
I'm not sure we already using the same thing for [Tag](https://github.com/JhonnyL/symfony/blob/calls-support-for-debug-container/src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php#L225) and [Autowiring Type](https://github.com/JhonnyL/symfony/blob/calls-support-for-debug-container/src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php#L194) and if we use `Calls` we should list them in the same row, did you see how it [looks](https://cloud.githubusercontent.com/assets/1104667/14232141/d09982ac-f99e-11e5-9432-4d97eb7db05b.png) ?